lint/linter/test-mdn-urls.ts

```diff
@@ -72,7 +109,24 @@ export const processData = (
         actual: data.mdn_url,
         expected: '',
       });
+    } else if (slugByPath.has(path)) {
```

Comment:
We don't want to overwrite links with fragments, so we need to add this check here: `!data.mdn_url.includes('#')`

Reply:
Fixed in [036c74a](https://github.com/mdn/browser-compat-data/pull/25563/commits/036c74a1d18f9d6d352e80cc72572eb13e43513a).